lua/lvim/config/defaults.lua

```diff
@@ -24,6 +24,35 @@ return {
     -- use config.lua for this not put here
   },
 
+  lazy = {
+    opts = {
+      install = {
+        missing = true,
+        colorscheme = { colorscheme, "lunar", "habamax" },
```

Comment:
```suggestion
        colorscheme = { "lunar", "habamax" },
```
this global doesn't exist, and at this point user's config isn't loaded yet